### PR TITLE
added velocity loss

### PR DESCRIPTION
**Pull request by HaKai-2062**

### CODE REVIEW

**Review by Rex109 — COMMENTED** [review]
I think there's a big issue with this... seems like it's triggering even if my velocity is perfectly stable and/or 0. plus i need you to leave the default ini settings to their default value and just add the new settings, other than this stuff it's a super-duper idea to add to the velometer!

**Comment by HaKai-2062** [comment]
ahhh oki i can change values to default but changing to velocityLoss color on consistent/stable speed was intended and dont you think it should work like that? 🤔

**Comment by HaKai-2062** [comment]
I can make it not trigger on stable velocity if u want...
Just my opinion that velocity loss color on stable velocity would look cool

**Comment by Rex109** [comment]
> ahhh oki i can change values to default but changing to velocityLoss color on consistent/stable speed was intended and dont you think it should work like that? 🤔

Honestly i've never seen something like that, maybe i'm wrong but then it will beome a velocity gain color change instead of velocity loss

**Comment by HaKai-2062** [comment]
yea in codjumper atleast that's how it usually works. Stable velocity and 0 is velocity loss color
i can just rename velocity color to velocity gain color and then should be good ^^

File: iw3velometer/dllmain.cpp
Comment (Rex109) [inline iw3velometer/dllmain.cpp]: Here

File: iw3velometer/dllmain.cpp
Comment (Rex109) [inline iw3velometer/dllmain.cpp]: Here

File: iw3velometer/dllmain.cpp
Comment (Rex109) [inline iw3velometer/dllmain.cpp]: Here

File: iw3velometer/dllmain.cpp
Comment (Rex109) [inline iw3velometer/dllmain.cpp]: Here

File: iw3velometer/dllmain.cpp
Comment (Rex109) [inline iw3velometer/dllmain.cpp]: Here

File: iw3velometer/dllmain.cpp
Comment (Rex109) [inline iw3velometer/dllmain.cpp]: Here

**Review by Rex109 — CHANGES_REQUESTED** [review]
Seems good now... the only thing i would ask you before merging is to fix the indention of some of the code, after that i'll give it a test and i'll let you know if i'll merge it or not.

**Review by Rex109 — APPROVED** [review]
Perfect! I'll merge the changes!

**Comment by Rex109** [comment]
And thanks for the font crash fix <3

**Pull request merged**
